webapp/configs/snapcraft.py

```diff
@@ -2,4 +2,5 @@
 
 WEBAPP_CONFIG = {"LAYOUT": "_layout.html", "STORE_NAME": "Snap store"}
 
+FEATURED_BANNERS_ENABLED = os.getenv("FEATURED_BANNERS_ENABLED", "false")
```

Comment:
How do you feel about doing something like:
```
_featured_banners_enabled = os.environ.get("FEATURED_BANNERS_ENABLED", default="false")
FEATURED_BANNERS_ENABLED = _featured_banners_enabled.lower() in {"1", "t", "true"}
```

Reply:
It's perfect, thanks! I forgot that this config file is just a python script, so I can parse the value in place instead of the views themselves.